src/docs/testing/debugging.md

```diff
@@ -222,6 +222,50 @@ If you use Travis or Cirrus for testing, increase the number of
 available file handlers that they can open by adding the same line to
 flutter/.travis.yml, or flutter/.cirrus.yml, respectively.
 
+### Widgets marked const that should be equal to each other, aren't
+
+In debug mode, you may find that two `const` widgets that should to all
+appearances be equal (because of Dart's constant deduplication) are not.
+
+For example, this code should print 1:
+
+```dart
+print(<Widget>{ // this is the syntax for a Set<Widget> literal
```

Comment:
@Sfshaza2 any idea why this isn't passing the syntax checks? do i have to declare that i'm importing something or...?

Reply:
Ian, I'm not sure how code checks work on flutter/flutter, but on flutter/website, the code must be a full example that can pass analysis without any other code. This is why we use the skip tag so liberally.

Reply:
ah ok, will skip then. thanks.